src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/Telemetry/VsTelemetryService.cs

```diff
@@ -15,6 +15,18 @@ internal class VsTelemetryService : ITelemetryService
         private const string PropertyPrefix = "VS.ProjectSystem.Managed.";
 
         /// <summary>
+        /// Post an event with the event name. This event will be correlated with the Project Telemetry Correlation asset.
```

Comment:
`This event will be correlated with the Project Telemetry Correlation asset.`. This statement is no longer true. I forgot to remove them when I made the changes, my bad.

We dont create any assets. So if possible can you remove that statement from all the doc comments? as a follow-up commit or PR?